src/content/public goods funding.md

```diff
@@ -0,0 +1,287 @@
+---
+title: Public goods funding 
+description: An overview of public goods funding solutions on Ethereum
+lang: en
+template: use-cases
+emoji: 🌎
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Public goods are nonexcludable and nonrivalrous and benefit everyone
+summaryPoint2: Traditional models of funding public goods are inefficient and encourage coordination failures
+summaryPoint3: Public goods markets on Ethereum make it easier to fund and develop critical digital public goods
+---
+ 
+Public goods are commodities and services provided to all members of a society (without the possibility of direct profit) usually by government or private organizations. While the creation of public goods is critical to societal progress and wellbeing, traditional models of coordinating to fund public goods suffer problems that limit their efficiency and effectiveness. This makes it difficult to build useful public goods and results in worse outcomes that affect everyone. 
```

Comment:
> (without the possibility of direct profit)

Does something making a profit make it **not** a public good? I'd argue not. To me, the main thing is that a public good isn't exclusionary (i.e. pay me money to use features) or rivalrous (me using it stops you from using it).

Reply:
> Or that the outcomes from a lack of public goods is worse because of the lacking funding.

This is what I had in mind.

Reply:
Changed the text now to add some clarity,